addons/hr_holidays/models/hr_leave_accrual_plan_level.py

```diff
@@ -53,6 +53,7 @@ class AccrualPlanLevel(models.Model):
         ('monthly', 'Monthly'),
         ('biyearly', 'Twice a year'),
         ('yearly', 'Yearly'),
+        ('allocation_start_date', 'At Allocation Start Date'),
```

Comment:
This new mechanism deserves a to be tested in ``addons/hr_holidays/tests/test_accrual_allocations.py``